UefiPayloadPkg/Library/PlatformBootManagerLib/PlatformBootManager.c

```diff
@@ -1111,6 +1128,17 @@ WarnIfFirmwareUpdateMode (
       NULL
       );
 
+  //
+  // Create volatile runtime variable so applications can detect FUM
+  //
+  Status = gRT->SetVariable (
+      L"FirmwareUpdateMode",
+      &gDasharoSystemFeaturesGuid,
+      EFI_VARIABLE_RUNTIME_ACCESS,

```

Comment:
Is EFI_VARIABLE_BOOTSERVICE_ACCESS not needed? iPXE does not operate at Runtime and VariableServiceQueryVariableInfo seems to require both BS and RT if RT is set

Reply:
Yes, it is needed. Page 215 of [UEFI Specification 2.10](https://uefi.org/sites/default/files/resources/UEFI_Spec_2_10_Aug29.pdf):

```
• Runtime access to a data variable implies boot service access. Attributes that have
EFI_VARIABLE_RUNTIME_ACCESS set must also have EFI_VARIABLE_BOOTSERVICE_ACCESS
set. The caller is responsible for following this rule.
```

Reply:
Fixed in https://github.com/Dasharo/edk2/compare/c4af5bac31238439de19e432e57eb3d0c5481f38..813a25418b2c3b636ea4e25765a705d3b3d29498

Reply:
It is sufficient to link just the commit hash addressing the review (if it is just a single commit)